<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>425</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Calcular EI</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>589</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Selecione o arquivo de temperaturas</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>351</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>30</y>
      <width>27</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxDecimal">
    <property name="geometry">
     <rect>
      <x>136</x>
      <y>80</y>
      <width>80</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>.</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>,</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>136</x>
      <y>60</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Decimal</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Separador</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxSep">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Espa√ßo</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>;</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>,</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="btnCalcular">
    <property name="geometry">
     <rect>
      <x>309</x>
      <y>250</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Calcular EI</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>281</width>
      <height>181</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnAbrir">
    <property name="geometry">
     <rect>
      <x>309</x>
      <y>80</y>
      <width>101</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Abrir</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxCSV">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>180</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Salvar CSV</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxHTML">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>120</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Salvar HTML</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxPDF">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>150</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Salvar PDF</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxSVG">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>210</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Salvar SVG</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
